src/lib/code-sample/javascript.ts

```diff
@@ -0,0 +1,18 @@
+import { camelCase } from 'change-case'
+
+import type { CodeSampleDefinition } from './schema.js'
+
+export const createJavascriptRequest = (
+  request: CodeSampleDefinition['request'],
+): string => {
+  const parts = request.path.split('/')
+  return `await seam.${parts.map((p) => camelCase(p)).join('.')}(${JSON.stringify(request.parameters)})`
```

Comment:
`path` starts with a slash, so when we `.split('/')` it, there'll always be an empty string as a first element which on `.join('.')` will get us a leading `.`.

```suggestion
  return `await seam${parts.map((p) => camelCase(p)).join('.')}(${JSON.stringify(request.parameters)})`
```